test_scripts/API/VehicleData/UnsubscribeVehicleData/011_UnsubscribeVD_for_2_apps_Expected_Disconnect_different_data.lua

```diff
@@ -0,0 +1,64 @@
+---------------------------------------------------------------------------------------------------
+---------------------------------------------------------------------------------------------------
+-- Issue: https://github.com/smartdevicelink/sdl_core/issues/2283
+--
+-- Description: Check that SDL processes UnsubscribeVehicleData RPC for two Apps with different <vd_param> parameters
+--  during an unregistration of App
+--
+-- Preconditions:
+-- 1) SDL and HMI are started
+-- 2) Two apps are registered and activated
+-- 3) SubscribeVehicleData, UnsubscribeVehicleData RPCs and <vd_param> parameter are allowed by policies
+-- 4) Apps are subscribed to <vd_param> parameter data
+--
+-- In case:
+-- 1) App_1 sends UnregisterAppInterface request to SDL
+-- SDL does:
+-- a) send UnsubscribeVehicleData request to HMI during the unregistration of App1
+-- 2) App_2 sends UnregisterAppInterface request to SDL
+-- SDL does:
+-- a) send UnsubscribeVehicleData request to HMI during the unregistration of App2
+---------------------------------------------------------------------------------------------------
```

Comment:
Since we have 2 VD parameters in this test I would add more details in description which app is subscribed to which parameter. And also which VD parameter should be in `UnsubscribeVehicleData` requests.
As we did in other `*_different_data` scripts.